### ISSUE TITLE
No focus ring around button based components with material ui

### ISSUE THREAD

**Issue by Pajn** [lusakasa/saka-key#153]
I created an issue at Material ui https://github.com/mui-org/material-ui/issues/9308 but later realized the problem cam by my usage of Saka key.

Is this an issue with Saka key or with Material ui?
If it is an issue with Material UI, does Saka key provide a way to know that a keyboard focus has occurred by using Saka key?

**Comment by Pajn** [comment]
I did some debugging and found that the keyup event on window is never fired when Saka is active.
Material UI uses this event to diffrentiate keyboard interactions from other interactions.

See the code:
https://github.com/mui-org/material-ui/blob/1bec6cf156e08a1f5896f9e984322abf4e29d4c7/packages/material-ui/src/ButtonBase/focusVisible.js#L71

The `handleKeyUpEvent` is never called. I have debugged Material UI and found that `listenForFocusKeys` is called with the actual window object both normally and when Saka key is active, so I can only conclude that something Saka does prevents keyup events from firing on the window object.
